libautoscoper/src/Trial.cpp

```diff
@@ -78,6 +153,62 @@ namespace xromm
     vector<string> renderResolution;
     vector<string> optimizationOffsets;
 
+    // Parse the file
+    parseFile(filename, &mayaCams, &camRootDirs, &volumeFiles, &voxelSizes, &volumeFlips, &renderResolution, &optimizationOffsets);
+
+    // Modifiy the relative paths to be absolute paths.
+    fs::path configLocation = fs::path(filename).parent_path();
+    fs::path tmpPath;
+    for (size_t i = 0; i < mayaCams.size(); ++i) {
+      tmpPath = fs::path(mayaCams[i]);
+      if (!tmpPath.is_absolute()) {
+				mayaCams[i] = (configLocation / mayaCams[i]).string();
+			}
+		}
+    for (size_t i = 0; i < camRootDirs.size(); ++i) {
+			tmpPath = fs::path(camRootDirs[i]);
+      if (!tmpPath.is_absolute()) {
+        camRootDirs[i] = (configLocation / camRootDirs[i]).string();
+      }
+		}
+    for (size_t i = 0; i < volumeFiles.size(); ++i) {
+      tmpPath = fs::path(volumeFiles[i]);
+      if (!tmpPath.is_absolute()) {
+        volumeFiles[i] = (configLocation / volumeFiles[i]).string();
+      }
+    }
```

Comment:
New code should be indented using spaces where one tab is equivalent to 2 spaces.